### PR TITLE
Module 1: Capstone project. 

### PR DESCRIPTION
**Pull request by svitalis123**
## Capstone Project.
**In this project I accomplished**
- Created a branch.
- Developed `Mobile` version and `Desktop` version.
- Implemented speaker page `dynamically`.
- [Link to explanation of my project](https://www.loom.com/share/7e9e8818142f41cab89b3e555ae6ce59).

### CODE REVIEW

File: app/app.css [hunk 1]
```diff
@@ -0,0 +1,662 @@
...
+}
+
+.container_sectionc_header h2 {
+  font-size: x-large;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.sectionb {
+  background: url(https://media.istockphoto.com/photos/modern-brick-wall-3d-rendering-picture-id1140265911?b=1&k=20&m=1140265911&s=170667a&w=0&h=cLoyrHySMr6agb3jd7kjL_KQZwW4c-JdgpCeFu_r81s=) 100% 100% no-repeat;
```
Comment (m05e5) [inline app/app.css]: - [ ] For the main program section, you are required to use the same background color as the template `#272A31`

File: app/app.css [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+
+.container_sectionb_columndivs {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  gap: 1rem;
+}
+
+.container_sectionb_columndivs button {
+  background: #f46139;
```
Comment (m05e5) [inline app/app.css]: - [ ] This is the orange recommended `#EC5242`

File: app.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    <div class="container_sectionc_speakers_speaker2_image_container">
+    </div>
+    <div class="container_sectionc_speakers_speaker2_content">
+        <h3>Chris Mburugu</h3>
+        <p class="p1">Registrar at Co-operative University.</p>
+        <div class="speaker_line"></div>
+        <p> Renowned for excellent public speaking, an expert in public relations, hosted at TED talk in 2021.</p>
+    </div>
+  </div>
+</div>
```
Comment (m05e5) [inline app.js]: - [ ] Hi. As mentioned in the requirements, you are required to create the speaker section dynamically. So, you need to have a variable with the list of objects(speakers) with the required info (name, role, description, picture) then do the function that loop through the list of speakers and  create the HTML when the page loads
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @svitalis123  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Hi. You are required to have Readme file following microverse template with an `Acknowledgements` section to your readme and appropriate credit to **Cindy Shin** the author of the original design.
- [ ] As part of the requirement you have to deploy your app using Github Page and add the link of your live demo in your Readme.md file
- [ ] You did a great job with the nav bar. Just, you are required to have the nav links at the right so I'll recommend reducing the gap between links and moving them to the right
image

- [ ] Nice footer, But you are required for this project to stick to the design as much as possible so I'll recommend putting your logo at the right and some text at the left

**Your design**
image
**Requirement**
image
- [ ] _Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_m05e5_


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @svitalis123    , 

Your project is complete and design is beautiful 👍 !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

- [ Optional] Please try to address the previous code reviewer Suggestion for  deploying your app using GitHub Page and add the link of your live demo in your Readme.md file. Your Link is broken. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by svitalis123** [comment]
thank you for the feedback sir

    On Wednesday, September 14, 2022 at 01:57:46 PM GMT+3, Caleb Moses HYMELE T. ***@***.***> wrote:  
 
 


@m05e5 requested changes on this pull request.

Required Changes ♻️ ⚠️ 🚦
 
Hi @svitalis123 ,

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



Good points 👍
 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️
 
Aspects to improve ♻️
    
   -  Hi. You are required to have Readme file following microverse template with an Acknowledgements section to your readme and appropriate credit to Cindy Shin the author of the original design.
   -  As part of the requirement you have to deploy your app using Github Page and add the link of your live demo in your Readme.md file
   -  You did a great job with the nav bar. Just, you are required to have the nav links at the right so I'll recommend reducing the gap between links and moving them to the right


   
   -  Nice footer, But you are required for this project to stick to the design as much as possible so I'll recommend putting your logo at the right and some text at the left

Your design

Requirement

   
   -  Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

m05e5

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using this form.

In app/app.css:
> +.container_sectiona_section_lower_content b {
+  font-size: x-large;
+}
+
+.container_sectionc_header {
+  padding: 2rem;
+}
+
+.container_sectionc_header h2 {
+  font-size: x-large;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.sectionb {
+  background: url(https://media.istockphoto.com/photos/modern-brick-wall-3d-rendering-picture-id1140265911?b=1&k=20&m=1140265911&s=170667a&w=0&h=cLoyrHySMr6agb3jd7kjL_KQZwW4c-JdgpCeFu_r81s=) 100% 100% no-repeat;
   
   -  For the main program section, you are required to use the same background color as the template #272A31

In app/app.css:
> +.container_sectionc_speakers {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 0.5rem;
+}
+
+.container_sectionb_columndivs {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  gap: 1rem;
+}
+
+.container_sectionb_columndivs button {
+  background: #f46139;
   
   -  This is the orange recommended #EC5242

In app.js:
> +<div class="container_sectionc_speakers">
+  <div class="container_sectionc_speakers_speaker1">
+    <div class="container_sectionc_speakers_speaker1_image_container">
+    </div>
+    <div class="container_sectionc_speakers_speaker1_content">
+        <h3>Kelvin Kimathi</h3>
+        <p class="p1">Co-operative engineer in IT</p>
+        <div class="speaker_line"></div>
+        <p>Kelvin Transformed IT in the University and published his book Infinite IT</p>
+    </div>
+  </div>
+  <div class="container_sectionc_speakers_speaker2">
+    <div class="container_sectionc_speakers_speaker2_image_container">
+    </div>
+    <div class="container_sectionc_speakers_speaker2_content">
+        <h3>Chris Mburugu</h3>
+        <p class="p1">Registrar at Co-operative University.</p>
+        <div class="speaker_line"></div>
+        <p> Renowned for excellent public speaking, an expert in public relations, hosted at TED talk in 2021.</p>
+    </div>
+  </div>
+  <div class="container_sectionc_speakers_speaker1 eliminate">
+    <div class="container_sectionc_speakers_speaker1_image_container">
+    </div>
+    <div class="container_sectionc_speakers_speaker1_content">
+        <h3>Kelvin Kimathi</h3>
+        <p class="p1">Co-operative engineer in IT</p>
+        <div class="speaker_line"></div>
+        <p>Kelvin Transformed IT in the University and published his book Infinite IT</p>
+    </div>
+  </div>
+  <div class="container_sectionc_speakers_speaker2 eliminate">
+    <div class="container_sectionc_speakers_speaker2_image_container">
+    </div>
+    <div class="container_sectionc_speakers_speaker2_content">
+        <h3>Chris Mburugu</h3>
+        <p class="p1">Registrar at Co-operative University.</p>
+        <div class="speaker_line"></div>
+        <p> Renowned for excellent public speaking, an expert in public relations, hosted at TED talk in 2021.</p>
+    </div>
+  </div>
+</div>
   
   -  Hi. As mentioned in the requirements, you are required to create the speaker section dynamically. So, you need to have a variable with the list of objects(speakers) with the required info (name, role, description, picture) then do the function that loop through the list of speakers and create the HTML when the page loads

**Pull request merged**